cli/azd/pkg/commands/pipeline/pipeline_manager.go

```diff
@@ -273,8 +287,17 @@ func (manager *PipelineManager) Configure(ctx context.Context) error {
 
 	if doPush {
 		err = manager.pushGitRepo(ctx, currentBranch)
+		if err == nil {
+			gitRepoInfo.pushStatus = true
+		}
+		err = manager.ScmProvider.postGitPush(
+			ctx,
+			gitRepoInfo,
+			manager.PipelineRemoteName,
+			currentBranch,
+			inputConsole)
 		if err != nil {
-			return err
+			return fmt.Errorf(" post git push hook : ")
```

Comment:
Is this missing formatting of the inner error?

Reply:
done